package.json

```diff
@@ -140,12 +140,12 @@
     "shelljs": "0.8.3",
     "shipjs": "0.16.0",
     "typescript": "3.7.5",
-    "webpack": "4.41.5"
+    "webpack": "4.42.0"
   },
   "bundlesize": [
     {
       "path": "./dist/instantsearch.production.min.js",
-      "maxSize": "63.5 kB"
+      "maxSize": "63.97 kB"
```

Comment:
do you know what caused this regression? It isn't big, but still surprising

Reply:
Some `dependencies` were updated and the lock file was regenerated. Since the UMD relies on these locked `dependencies`, their latest versions are a bit bigger. We should be careful with regenerating the lock file. I think we should revert that.